builds/e2e/e2e.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
         # We use a self-hosted agent for this job, so we need to handle
         # cleanup of root-owned files ourselves (anything created during
         # `sudo dotnet vstest ...`)
-        sudo git clean -ffdx .
+        sudo git clean -ffdx . | true
```

Comment:
I believe you want to use logical OR `||` here, not pipe `|`. We use logical OR for this pattern in our other scripts. Also, the pipe will swallow stdout from the `git clean` command...I think we'd like to see the command's output.